### PR TITLE
Adding an apartment-update-details view

### LINKED ISSUE #97: Add an apartment-update-details view
- Add an apartment-update-details view.
- Add Form classes to handle the details update.
- Add a template to update the details.

### PR DESCRIPTION
**Pull request by tamirho**
- Adding apartment-update-details view.
- Adding two Form classes to handle the details update.
- Adding a template to update the details.

Fix #97 

**Note:**
The template needs to be redesigned in the future

### CODE REVIEW

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Please add tests for the view

**Comment by tamirho** [comment]
> Please add tests for the view

Please review again

File: apartments/tests.py [hunk 1]
```diff
@@ -3,43 +3,46 @@
 import pytest
 
 
+@pytest.fixture
```
Comment (Omeramsc) [inline apartments/tests.py:6]: All of those fixtures exists also in `contacts/tests.py`, check if there is a way to use the same fixtures without creating duplicates.
I can see that it's not originated in this PR, so I'll approve it regardless, but create an issue and another PR to fix it (also check if it repeats in other places)

**Review by Omeramsc — APPROVED** [review]

File: apartments/tests.py (hunk 1 above)
Comment (tamirho) [reply apartments/tests.py:6]: Ok, I will check it and fix it in another PR.

**Review by Micha546 — COMMENTED** [review]
LGTMicha

**Review by DanielMy777 — COMMENTED** [review]
Great!!

**Review by simzacks — APPROVED** [review]

**Pull request merged**